### ISSUE TITLE
.obj rendering issue

### ISSUE THREAD

**Issue by cpheinrich** [indianajohn/web-geo-viewer#1]
Awesome tool! I think this will be very helpful to me workflows with mesh data. I just tried this out on an .obj file I recently generated, and unfortunately it does not seem to be rendering properly. The obj file can be downloaded from here: https://drive.google.com/file/d/1yM1O06x16k3jFDOO5kbmxN2TK_c9ureJ/view?usp=sharing

When rendering in meshlab this looks like:
<img width="854" alt="Screen Shot 2020-06-22 at 6 01 29 PM" src="https://user-images.githubusercontent.com/12475454/85349110-6d59c700-b4b2-11ea-9fae-89ae17d8ff47.png">

However in the web geo viewer it looks like:
![Screen Shot 2020-06-22 at 6 00 55 PM](https://user-images.githubusercontent.com/12475454/85349117-73e83e80-b4b2-11ea-8740-4b9258122a53.png)

Please let me know if I am doing anything wrong, of this is an issue that needs to be corrected

**Comment by indianajohn** [comment]
I figured out what the issue is. The file you uploaded contains vertex groups, which is not supported by the rust-3d crate's OBJ parser and therefore isn't supported by the fork in this crate.

I'll come up with a fix soon.

**Comment by indianajohn** [comment]
I fixed this and deployed it to the site.

This is what I get when I render that OBJ now.

![Screenshot 2020-06-22 at 8 43 31 PM](https://user-images.githubusercontent.com/3815530/85358782-6db18c80-b4c9-11ea-9a11-7d56c502b0d9.png)

There were 2 issues:
* Some of the faces in that OBJ were erroneous: they contained the same index twice. I made the loader more robust to these faces; it now simply skips them.
* The MTL file attached contains an = where it shouldn't according to the OBJ file specification. It says "Kd = 0.5 0.5 0.5" when the appropriate line would be "Kd 0.5 0.5 0.5". I changed the parser to skip "=" on such lines when it encounters such a line, since data in the real world is rarely perfect.

For the changes to take effect for you, you might need to go to the page, go to upper-right hamburger menu (on chrome) -> developer tools, then click and hold the refresh button and click "hard reload."
